src/options.c

```diff
@@ -221,6 +221,24 @@ options_load_from_elektra(
             break;
         }
     }
+    
+    // Easing mode
+    ElektraEnumFadeEasing easingMode = elektraGetFadeEasing(elektra);
+    switch (easingMode) {
+      case ELEKTRA_ENUM_FADE_EASING_LINEAR:
+	options->easing_mode = EASING_MODE_LINEAR;
```

Comment:
Why didn't you directly use Elektra's enum?

Reply:
I was probably influenced by my "implementation without Elektra".

Thanks for the hint, I've refactored this!